metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -358,7 +359,16 @@ def _get_dataset_column_median(
         if not self.config.include_field_median_value:
             return
         try:
-            column_profile.median = str(self.dataset.get_column_median(column))
+            if self.dataset.engine.dialect.name.lower() == "snowflake":
+                column_profile.median = str(
+                    self.dataset.engine.execute(
```

Comment:
Any reason this would fail and we would want to try catch here?

Reply:
This code is in a try-catch block?